src/test/resources/archives/case_claim_with_multi_select/modules-0/forms-0.xml

```diff
@@ -0,0 +1,120 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
```

Comment:
since we don't care about the form content for these test it would be nice to simplify these to the most basic form